app/tasks/maintenance.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+global $cli;
+
+use Utopia\App;
+use Utopia\CLI\Console;
+
+$cli
+    ->task('maintenance')
+    ->desc('Schedules maintenance tasks and publishes them to resque')
+    ->action(function () {
+        $interval = App::getEnv('_APP_MAINTENANCE_INTERVAL', '');
+
+        for($i = 0; $i <= 10; ++$i){
+            Console::log('Starting the maintenance worker every '.$interval.' seconds');
+            sleep($interval);
```

Comment:
@eldadfux is the `Console::loop` function available to be consumed here ?

Reply:
Yes. Update the composer dependency to the new utopia/cli:0.8.0 version and it will be available. Appwrite container rebuild is required.

Reply:
Done